packages/react-components/src/components/LocalVideoTile.tsx

```diff
@@ -110,19 +146,73 @@ export const _LocalVideoTile = React.memo(
     useLocalVideoStreamLifecycleMaintainer(localVideoStreamProps);
 
     /* @conditional-compile-remove(spotlight) */
-    const spotlightBorder = useMemo(
-      () => (
-        <Stack
-          className={mergeStyles({
-            position: 'absolute',
-            height: '100%',
-            width: '100%',
-            zIndex: 100,
-            border: `0.25rem solid ${theme.palette.black}`
-          })}
-        />
-      ),
-      [theme.palette.black]
+    const contextualMenuProps = useVideoTileContextualMenuProps({
+      participant: { userId: userId ?? '' },
+      strings: { ...strings },
+      spotlightedParticipantUserIds,
+      isSpotlighted,
+      onStartSpotlight,
+      onStopSpotlight,
+      maxParticipantsToSpotlight,
+      myUserId: userId
+    });
+
+    const videoTileContextualMenuProps = useMemo(() => {
+      /* @conditional-compile-remove(spotlight) */
+      if (menuKind !== 'contextual' || !contextualMenuProps) {
+        return {};
+      }
+      /* @conditional-compile-remove(spotlight) */
+      return {
+        contextualMenu: contextualMenuProps
+      };
+      return {};
+    }, [
+      /* @conditional-compile-remove(spotlight) */ contextualMenuProps,
+      /* @conditional-compile-remove(spotlight) */ menuKind
+    ]);
+
+    const videoTileStyles = useMemo(() => {
+      /* @conditional-compile-remove(spotlight) */
+      if (isSpotlighted) {
+        return concatStyleSets(
+          {
+            root: {
+              '&::after': {
```

Comment:
Do we need to do this through the after? are we able to apply an outline to do this instead? Take a look at the BackgroundEffectPicker to see how we do that as well https://github.com/Azure/communication-ui-library/blob/main/packages/react-components/src/components/VideoEffects/VideoEffectsItem.styles.tsx

Reply:
I will try outline and see. Thanks